examples/with-tailwind/packages/ui/tailwind.config.js

```diff
@@ -1,3 +1,6 @@
 const config = require("tailwind-config/tailwind.config.js");
 
-module.exports = config;
+module.exports = {
+  ...config,
+  prefix: 'ui-',
```

Comment:
You'll need to update all the classes used by UI to use this prefix as well (check the preview and you'll see that some of the styles are missing from the button)

Reply:
@tknickman - Thanks for reviewing! I'm looking into it for a better solution, as only adding a prefix may not be ideal for DX (i.e. too many classNames have to add the prefix).

Reply:
@tknickman - pushed fix. Pls review. Thx.